src/onegov/town6/views/political_business.py

```diff
@@ -108,7 +108,7 @@ def view_political_businesses(
             text=request.translate(text) +
                 f' ({count_per_status[status]})',
             active=status in self.status,
-            url=request.link(self.for_filter(s=status)),
+            url=request.link(self.for_filter(status=status)),
```

Comment:
```suggestion
            url=request.link(self.for_filter(status=[status])),
```
I think this should be like this now.